sway/input/cursor.c

```diff
@@ -513,6 +515,7 @@ static void handle_touch_up(struct wl_listener *listener, void *data) {
 		}
 	} else {
 		wlr_seat_touch_notify_up(wlr_seat, event->time_msec, event->touch_id);
```

Comment:
I think this `wlr_seat_touch_notify_up` belongs in the seatop, now that we have one.

Reply:
I moved those notification and focus action in the seatop